couchbase_test.go

```diff
@@ -0,0 +1,694 @@
+package couchbase
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	docker "github.com/hashicorp/vault/helper/testhelpers/docker"
+	"github.com/hashicorp/vault/sdk/database/dbplugin"
+	"github.com/ory/dockertest"
+	dc "github.com/ory/dockertest/docker"
+)
+
+var containerInitialized bool = false
+var cleanup func() = func() {}
+var pre_6dot5 = false // check for Pre 6.5.0 Couchbase
```

Comment:
Rename this to pre6Dot5 for consistency with Go naming conventions.

Reply:
Done.